examples/rollup.config.js

```diff
@@ -0,0 +1,32 @@
+import resolve from "@rollup/plugin-node-resolve";
```

Comment:
FYI, this from Rollup's docs:

```
# if you do not pass a file name, Rollup will try to load
# configuration files in the following order:
# rollup.config.mjs -> rollup.config.cjs -> rollup.config.js
rollup --config
```

So you can happily name this `rollup.config.mjs`, and even skip explicitly naming the config in the package.json.

Reply:
I've removed the name from the npm scripts but I like that the rollup.config.js file shows up in VSCode with it's own icon. Makes it super easy to find.